swift/ql/lib/codeql/swift/dataflow/FlowSummary.qll

```diff
@@ -10,7 +10,7 @@ class ArgumentPosition = DataFlowDispatch::ArgumentPosition;
 
 // import all instances below
 private module Summaries {
-  /* TODO */
+  private import codeql.swift.frameworks.Frameworks
```

Comment:
Is this import how we prevent re-evaluation of cached stages?

Reply:
Yeah, exactly. It may have been fine before (because all places that imported `FlowSummary` also imported `ExternalFlow`), but now it's more explicit.